psi4/driver/procrouting/scf_proc/scf_iterator.py

```diff
@@ -425,6 +428,9 @@ def scf_iterate(self, e_conv=None, d_conv=None):
                 if self.frac_performed_:
                     status.append("FRAC")
 
+                if incfock_performed:
```

Comment:
I notice that this if statement uses an incfock_performed variable determined via parameters in jk, while the if statements directly above this one use variables contained within the HF class (MOM_performed_ and frac_performed_). For consistency's sake, it might be better to move incfock_performed_ to the HF class and keep this if statement as "if self.infock_performed_:"

Reply:
Absolutely not.

1) Class variables are a necessary evil at best. (Good luck trying to figure out where the variable was last modified.) Local variables are preferred.
2) `incfock_performed_` isn't something that the HF object should know about. It doesn't mean anything unless you're using a direct algorithm.

Reply:
All right, that makes sense. I had mainly suggested it for consistency's sake; but if it is better this way, then there is no real need to change it.